app/classifier/task-nav.jsx

```diff
@@ -105,27 +112,26 @@ class TaskNav extends React.Component {
   }
 
   render() {
-    if (!this.props.annotation) {
-      return null;
-    }
+    const completed = !!this.props.classification.completed;
+
     const task = this.props.task ? this.props.task : this.props.workflow.tasks[this.props.workflow.first_task];
```

Comment:
On second thoughts, looks like the additional checks for the existence of `annotation` have addressed this.